crates/gpui/src/app.rs

```diff
@@ -341,12 +343,9 @@ impl SystemWindowTabController {
 
     /// Get all tabs in the same window.
     pub fn tabs(&self, id: WindowId) -> Option<&Vec<SystemWindowTab>> {
```

Comment:
This has become such a small helper that I'd consider removing it entirely. It's only called a few times and the majority of the time, we use `.position()` right after to find the index of tab with the right `WindowId`. Which could be done right away instead of using the helper first, thus removing an iteration through the values.